### DISCUSSION TITLE
GoPro Hero 8 - any way to delay start for timelapse photos?

### DISCUSSION THREAD

**Discussion by eboblin97** [gopro/labs#354] [General]
Hi,

We are looking to use a Go Pro Hero 8 - for timelapse photos - but can't find a way to delay the start for a few hours. 
We have use QR codes and found a way to delay the start for video however when trying this in timelapse mode it doesn't seem to work.

To clarify, we want to have the GoPro switched off for around 6 hours, and then to come on and collect timelapse photos until the battery dies. The camera will be underwater so we can't manually switch it on.

Thanks!

**Comment by dnewman-gpsw** [comment]
Share the command you have tried, as delayed start is a base Labs function.
